sympy/simplify/cse_main.py

```diff
@@ -445,7 +445,7 @@ def _find_opts(expr):
             _find_opts(e)
 
     # split muls into commutative
-    comutative_muls = set()
+    comutative_muls = OrderedSet()
```

Comment:
commutative -> commutative (I know that python doesn't care, but if we're going to spell out this much, let's go ahead and give it an M&M. :-))